snackRouters.test.js

```diff
@@ -0,0 +1,45 @@
+const request = require('supertest');
+const app = require('./index');
+
+const db = require('./data/dbconfig');
+
+beforeAll(async done => {
+    await db.seed.run();
+    done();
+})
+
+describe('snackRouters', () => {
+    describe('POST /snacks', () => {
+        test('should respond with 201', () => {
+            return request(app)
+                .post('/snacks')
+                .send({
+                    name: "Fruit"
+                })
+                .expect(201)
+        });
+
+        test('should respond with json', () => {
+            return request(app)
+                .post('/snacks')
+                .send({
+                    name: "Fruit"
+                })
+                .expect('Content-Type', /json/)
+        })
+    });
+
+    describe('DELETE /snack', () => {
```

Comment:
I like that you've wrapped each endpoint in their own `describe` - makes the code (and tests) really easy to read